crates/store/re_video/benches/video_load_bench.rs

```diff
@@ -0,0 +1,15 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+
+fn video_load(c: &mut Criterion) {
+    let video = include_bytes!("../../../../tests/assets/video/Big_Buck_Bunny_1080_10s_av1.mp4");
```

Comment:
hmmm do you really want to fail compilation only because the user didn't run `download_test_asset`? better to make this dynamic loading

awesome though that you're trying to make this _a thing_ and not just a throw-away/one-off benchmarking 👍

Reply:
oops